clients/python-apiserver-client/python_apiserver_client/params/templates.py

```diff
@@ -0,0 +1,137 @@
+import enum
+
+
+class TolerationOperation(enum.Enum):
+    Exists = "Exists"
+    Equal = "Equal"
+
+
+class TolerationEffect(enum.Enum):
+    NoSchedule = "NoSchedule"
+    PreferNoSchedule = "PreferNoSchedule"
+    NoExecute = "NoExecute"
+
+
+class Toleration:
```

Comment:
Are the concepts in this file covered by some general-purpose K8s Python lib? It seems like "toleration" is classic affinity and anti-affinity. Is there a standard construct for "templates"?

Reply:
It is covered by K8 documentation